includes/admin/class-admin.php

```diff
@@ -1162,6 +1179,42 @@ private function check_deprecation_warnings() {
             }
         }
 
+		/**
+		 * Check old inline fontawesome setting.
+		 * If it was enabled by default, we show an admin notice.
+		 * On dismiss we keep the setting.
+		 *
+		 * @since x.x
+		 */
+		private function check_font_awesome_setting() {
+			if ( ! current_user_can( 'install_plugins' ) || ! WPBDP_App_Helper::is_bd_page() ) {
+				return;
+			}
+
+			if ( ! wpbdp_get_option( 'enqueue-fontawesome-styles', false ) ) {
+				return;
+			}
+
+			if ( is_plugin_active( 'font-awesome/index.php' ) ) {
```

Comment:
Let's switch all these checks to be more universal like we did in the other PR. We should check the styles instead of a single plugin.

Reply:
Moved the checks to the helper class